paymentsheet/src/test/java/com/stripe/android/paymentsheet/PaymentSheetViewModelTest.kt

```diff
@@ -1356,7 +1357,8 @@ internal class PaymentSheetViewModelTest {
         viewModel.registerFromActivity(DummyActivityResultCaller(), lifecycleOwner)
         lifecycleOwner.handleLifecycleEvent(Lifecycle.Event.ON_DESTROY)
 
-        verify(paymentLauncher.authenticatorRegistry).unregisterAuthenticator(any())
+        // unregisterAuthenticator should be called for each call to registerAuthenticator.
+        verify(paymentLauncher.authenticatorRegistry, times(2)).unregisterAuthenticator(any())
```

Comment:
Can we make this test validate the argument of `unregisterAuthenticator`? We want to make sure it’s unregistering `BlikAuthorize` and `UpiAwaitNotification`.